numba_cuda/numba/cuda/cudadrv/nvrtc.py

```diff
@@ -348,16 +336,10 @@ def compile(src, name, cc, ltoir=False):
         f"{os.path.join(cuda_include_dir, 'cccl')}",
     ]
 
-    nvrtc_version = nvrtc.get_version()
-    nvrtc_ver_major = nvrtc_version[0]
-
     cudadrv_path = os.path.dirname(os.path.abspath(__file__))
     numba_cuda_path = os.path.dirname(cudadrv_path)
 
-    if nvrtc_ver_major == 11:
-        numba_include = f"{os.path.join(numba_cuda_path, 'include', '11')}"
-    else:
-        numba_include = f"{os.path.join(numba_cuda_path, 'include', '12')}"
+    numba_include = f"{os.path.join(numba_cuda_path, 'include', '12')}"
```

Comment:
Might need additional logic for CUDA 13.

Reply:
It looks like all of these headers are for bfloat16 support. Is this something that we still need to vendor or can we get it from the Toolkit? I saw “ bfloat16 requires compute capability 8.0+ and CUDA version>= 12.0” which makes me think we should not need to vendor?

Can we delete the directory of CUDA 11 includes? https://github.com/gmarkall/numba-cuda/tree/cuda-13/numba_cuda/numba/cuda/include/11

Reply:
We vendor them to avoid having to discover the correct version of the headers (or assume their existence on the system).

I don't mind considering getting rid of the vendoring in the long term, but I think this causes no issue for now, and trying to rely on finding them on the users' system and getting it right in all cases will likely require a quantity of effort that I just don't think makes sense to prioritise with the massive queue of outstanding other items right now.

We can delete the CUDA 11 includes.